llama_index/readers/base.py

```diff
@@ -18,3 +22,30 @@ def load_langchain_documents(self, **load_kwargs: Any) -> List[LCDocument]:
         """Load data in LangChain document format."""
         docs = self.load_data(**load_kwargs)
         return [d.to_langchain_format() for d in docs]
+
+
+class PydanticBaseReader(BaseReader, BaseComponent):
+    """Serialiable Data Loader with Pydatnic."""
+
+    is_remote: bool = False
+
+    class Config:
+        arbitrary_types_allowed = True
+
+
+class LoaderConfig(BaseComponent):
```

Comment:
Sometimes I find it pretty confusing that we call it loader sometimes, reader other times, and connectors remaining times.

Reply:
Ranemed to `ReaderConfig` :)